src/workflows/template/validate.yml.tmpl

```diff
@@ -0,0 +1,23 @@
+name: Validate
+{{- if .Push -}}
+{{/* the Push workflow contains validate operation, so skip main branch */}}
+on:
+  push:
+    branches:
+      - '**'
+      - '!{{ .MainBranch }}'
```

Comment:
Ak sa generuje aj `push` akcia, tak pre `main` vetvu negenerujeme `Validate` akciu, bolo by to zbytocne duplicitne.